cristin-person/src/main/java/no/unit/nva/cristin/person/update/PersonPatchValidator.java

```diff
@@ -23,6 +35,7 @@ public static void validate(ObjectNode input) throws BadRequestException {
         validateFirstNameIfPresent(input);
         validateLastNameIfPresent(input);
         validateReservedIfPresent(input);
+        validateEmploymentsIfPresent(input);
```

Comment:
Add a private constructor since this is all static and the implicit constructor from Java should not be accessible:

@JacocoGenerated
private PersonPatchValidator() {
    // NO-OP
}